content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -25,7 +25,12 @@ It does not mean that there is a file named `kubeconfig`.
 
 {{% capture prerequisites %}}
 
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+To check that {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} is installed,
+run `kubectl version --client`. The kubectl version should be
+[within one minor version](/docs/setup/release/version-skew-policy/#kubectl) of your
```

Comment:
@sftim ,
The changes seem okay. I wonder if the `version-check` shortcode should be updated to
render (optionally) a `client` flag (and possibly the skew statement)?

Reply:
That's also a good idea!

Reply:
> @sftim ,
> The changes seem okay. I wonder if the `version-check` shortcode should be updated to
> render (optionally) a `client` flag (and possibly the skew statement)?

Good candidate for a follow up PR.